example/src/test/java/org/wordpress/android/fluxc/list/post/PostListDescriptorTest.kt

```diff
@@ -212,7 +213,23 @@ internal class PostListDescriptorTest(
                             ),
                             shouldHaveSameTypeIdentifier = true,
                             shouldHaveSameUniqueIdentifier = true
+                    ),
+                    // Different only author id
+                    PostListDescriptorTestCase(
+                            typeIdentifierReason = "Different onlyAuthorId should have same type identifiers",
```

Comment:
`onlyAuthorId` is not a valid name anymore, we can probably change it to:
```suggestion
                            typeIdentifierReason = "Different author filters should have same type identifiers",
```